pandas/tests/io/test_parquet.py

```diff
@@ -812,11 +812,15 @@ def test_write_with_schema(self, pa):
     def test_additional_extension_arrays(self, pa):
         # test additional ExtensionArrays that are supported through the
         # __arrow_array__ protocol
+
+        from pandas.core.arrays.string_arrow import ArrowStringDtype  # noqa: F401
+
         df = pd.DataFrame(
             {
                 "a": pd.Series([1, 2, 3], dtype="Int64"),
                 "b": pd.Series([1, 2, 3], dtype="UInt32"),
                 "c": pd.Series(["a", None, "c"], dtype="string"),
+                "d": pd.Series(["a", None, "c"], dtype="arrow_string"),
```

Comment:
this test is `@td.skip_if_no("pyarrow", min_version="0.15.0")` so safe to add here.

Reply:
maybe not `ImportError: pyarrow>=1.0.0 is required for PyArrow backed StringArray.`

Reply:
Yeah, this test has a minimum of 0.15. I would make a separate, dedicated test for ArrowStringArray